enhancements/autoscaling/provisioning-request-enablement.md

```diff
@@ -0,0 +1,337 @@
+* Configure the Cluster Autoscaler to enable the controller for ProvisioningRequests.
+
+### Non-Goals
+
+* Rename the ProvisioningRequest API type, group, or version.
+* Add an API field to the ClusterAutoscaler resource to allow configuration of the ProvisioningRequest controller.
+
+## Proposal
+
+1. Add the ProvisioningRequest custom resource definition to the manifest payload for the Cluster Autoscaler Operator.
+2. Modify the Cluster Autoscaler Operator to enable the `--enable-provisioning-requests` flag of the Cluster Autoscaler.
+3. Update the Cluster Autoscaler RBAC to allow reading and writing of ProvisioningRequests.
+
+### Workflow Description
+
+This feature is exposing an API resource type and does not provide the user with additional
+controls for configuring how the resource is consumed. There are two general workflows
+that users will encounter:
+
+**cluster administrator** is a human user responsible for deploying cluster-wide changes
+for users of the cluster.
+
+**application administrator** is a human user responsible for deploying an application in
+a cluster.
+
+**Example workflow 1**
+
+1. The cluster administrator configures the Cluster Autoscaler by creating a ClusterAutoscaler
+  resource and one or more MachineAutoscaler resources.
+2. The application administrator creates a manifest for a ProvisioningRequest resource in the
+  `openshift-machine-api` namespace and applies it with the `oc` command line tool.
+3. The Cluster Autoscaler reconciles the ProvisioningRequest and begins creating the
+  requested cluster capacity.
+
+**Example workflow 2**
+
+1. The cluster administrator configures the Cluster Autoscaler by creating a ClusterAutoscaler
+  resource and one or more MachineAutoscaler resources.
+2. The cluster administrator installs and configures Kueue for cluster users, with support
+  for ProvisioningRequest enabled.
+3. The application administrator submits their workload to Kueue for scheduling.
+4. At the appropriate time, Kueue creates a ProvisioningRequest resource in the `openshift-machine-api`
+  namespace to represent the workload and the Cluster Autoscaler reconiles the resource
+  and begins to create the requested cluster capacity.
+
+### API Extensions
+
+- Adds the Kubernetes custom resource definition for `provisioningrequests.autoscaling.x-k8s.io`
+  to all OpenShift installations where the Cluster Autoscaler Operator is also installed.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+Hypershift clusters utilize the Cluster Autoscaler by deploying it through a Hypershift controller
+instead of the Cluster Autoscaler Operator. As such, this API change should be deployed by a component
+associated with Hypershift to ensure it is available. For Hypershift, this enhancement
+proposes the following changes:
+
+* The hosted control plane operator will install the `provisioningrequests.autoscaling.x-k8s.io`
```

Comment:
Let's change this to clarify that this CRD will be installed as part of the CVO payload. For that to happen the CRD will need to include this annotation https://github.com/openshift/cluster-monitoring-operator/blob/18bb85225cd1ea3100ca9a45b26f456b2d681ee4/manifests/0000_50_cluster-monitoring-operator_01-namespace.yaml#L7

Reply:
ack, will do.